region/maridia/inner-pink/Below Botwoon Energy Tank.json

```diff
@@ -126,6 +126,24 @@
         }
       }
     },
+    {
+      "link": [1, 1],
+      "name": "Leave With Runway - Frozen Zoa",
+      "requires": [
+        "h_canFrozenEnemyRunway"
+      ],
+      "exitCondition": {
+        "leaveWithRunway": {
+          "length": 5,
+          "openEnd": 0
+        }
+      },
+      "note": [
+        "The Zoa needs to be at least 3 pixels higher than when it is frozen at standing height.",
+        "It may help to freeze one while standing to use as a reference, then jump and freeze a second while, noting that it will move up 1 more pixel once it thaws.",
```

Comment:
Freeze one facing left and still rising, then stand in a particular spot on the slope behind it, on the right.